README.md

```diff
@@ -1 +1,24 @@
 # hw01
+
+# INTRODUCTION
+My name is Nairi Brown. I am from Massachusetts. I also went to UChicago for undergrad. 
+At home, I have a wonderful dog named **Louie** who is adorable. 
+
+*see picture below*
+
+![](lou.jpeg)
```

Comment:
Your dog is very cute! Not to repeat our professor, but perhaps he could have been scaled down a little (not that I know how to do that...)

Reply:
Resizing images in Markdown/GitHub can be a bit tricky, so I would recommend resizing the original images if you decide to go down that route